app/Lib/lang.php

```diff
@@ -1060,6 +1060,7 @@
   ),
   
   // Errors
+  'er.500' =>         'Internal server error (500)',
```

Comment:
If we're going to start localizing HTTP codes, they should probably be grouped together, eg `er.http.500`.

Reply:
See the following comment (question) about er.http.401.